homeassistant/components/evohome/__init__.py

```diff
@@ -221,9 +251,116 @@ def _handle_exception(err) -> bool:
         broker.update, config[DOMAIN][CONF_SCAN_INTERVAL]
     )
 
+    setup_service_functions(hass, broker)
+
     return True
 
 
```

Comment:
Please add `@callback` decorator.

Reply:
Sorry @balloob Do you mean add it thus (my guess):
```python
@callback
def setup_service_functions( ...
```
... or:
```python
@callback
async def async_setup( ...
```

TBH, I am not sure why it is needed!

Reply:
OK, I did some reading - I think this is right.